src/ec/packages/ec-component-select/ecl-select.html.twig

```diff
@@ -42,6 +46,10 @@
 {% set _required_text = required_text|default('') %}
 {% set _optional_text = optional_text|default('') %}
 {% set _label = label|default('') %}
+{% set _multiple = multiple|default(false) %}
+{% set _multiple_placeholder = multiple_placeholder|default('') %}
+{% set _multiple_all_text = multiple_all_text|default('') %}
```

Comment:
but i don't see how this could depend on the twig template, @papegaill, the prop is mandatory, so i don't even think this is a critical bug, but i believe this is about the ECL vanilla component.

Reply:
it seems that by default `multiple_search_text` is an empty string, so removing the label by the knobs, the display shouldn't break like this, instead, we should have an empty input without a placeholder?

Reply:
So, i checked, @papegaill, the reason for this is a javascript error 
`Uncaught TypeError: this.selectAll.addEventListener is not a function`
occurring when there is no value for that prop, which just set the value of a custom attribute.
I agree that this should be better handled, but it's not about twig, we can't set a default for that prop, for instance, so this might be fixed on the ECL component, in case.
I created this ticket for ECL: https://citnet.tech.ec.europa.eu/CITnet/jira/browse/FRONT-1274

Reply:
ok thx for this.